recipe/session/sessmodels/models.go

```diff
@@ -24,6 +24,22 @@ import (
 	"github.com/supertokens/supertokens-golang/supertokens"
 )
 
+type TokenType string
+
```

Comment:
search for these terms in the project and make sure they don't exist (other than in recipeimplementation for legacy part):
- idrefreshtoken
- id-refres-token
- id refresh token